libc/src/unistd/pathconf_utils.h

```diff
@@ -0,0 +1,23 @@
+//===-- Implementation header for pathconf_utils ----------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_UNISTD_PATHCONF_UTILS_H
+#define LLVM_LIBC_SRC_UNISTD_PATHCONF_UTILS_H
+
+#include <unistd.h>
+
+namespace LIBC_NAMESPACE {
+
+LIBC_INLINE long filesizebits(const struct statfs &s);
```

Comment:
Remove `LIBC_INLINE`. You have separated definitions into a `.cpp` file then you cannot inline these symbols